apps/sms/js/thread_ui.js

```diff
@@ -2799,10 +2816,14 @@ var ThreadUI = {
       if (Settings.supportEmailRecipient) {
         items.push({
           l10nId: 'sendMMSToEmail',
-          method: function oMMS(param) {
-            ActivityPicker.sendMessage(param);
+          method: () => {
+            this.navigateToComposer({
+              activity: { number: email }
+            });
           },
-          params: [email]
+          // As we change panel here, we don't want to call 'complete' that
+          // changes the panel as well
+          incomplete: true
```

Comment:
Sorry I don't quite understand this part and I can't reproduce it...  Could you please explain more?